src/Uno.UI/UI/Xaml/Controls/Border/Border.iOSmacOS.cs

```diff
@@ -20,20 +20,8 @@
 
 namespace Microsoft.UI.Xaml.Controls;
 
-public partial class Border
+partial class Border
 {
-	partial void OnChildChangedPartial(UIElement previousValue, UIElement newValue)
-	{
-		previousValue?.RemoveFromSuperview();
-
-		if (newValue != null)
-		{
-			AddSubview(newValue);
-		}
-
-		UpdateBorder();
```

Comment:
Was this call not necessary on iOS and macOS?

Reply:
`BorderLayerRenderer` attaches itself to `LayoutUpdated` event, so I assume this is superfluous as it will happen anyway.

Reply:
Just to make sure, I think `AfterUpdateBorderPartial` will not be called when `LayoutUpdated` is fired. Only `BorderLayerRenderer.Update` is called. Want to make sure this is intended.

Reply:
I'd tend to think that `AfterUpdateBorderPartial` is still required as well.

Reply:
Yes, adjusted the code so that the logic happens inside `BorderLayerRenderer` itself, which simplifies things